generators/hls-example/run_hls.tcl

```diff
@@ -0,0 +1,8 @@
+open_project -reset proj_gcd_example
+add_files accel/HLSAccel.cpp
+set_top HLSAccelBlackBox
+open_solution -reset "solution1"
+set_part {xcvu9p-flgb2104-2-i}
```

Comment:
Why do you need a part number? This is FPGA specific so maybe the docs should mention this RTL that is made is specific to an FPGA (with part number listed here) but can be run in RTL sim.